packages/experiments/src/Foundation.ts

```diff
@@ -86,7 +86,7 @@ function getCustomizations<TViewProps, TStyleSet>(
 ): IStyleableComponentProps<TViewProps, TStyleSet, ITheme, ISchemeNames> {
   // TODO: do we want field props? should fields be part of IComponentOptions and used here?
   // TODO: should we centrally define DefaultFields? (not exported from styling)
-  const DefaultFields = ['theme', 'styles'];
+  const DefaultFields = ['theme', 'styles', 'styleVariables'];
```

Comment:
I need to resolve the TODO above. Think `styleVariables` will apply to all components like `theme` and `styles`?